### PR TITLE
WIP: Wg devsecops

### PR DESCRIPTION
**Pull request by HumairAK**
Depends on: https://github.com/open-services-group/community/pull/3 

I added onto the sig introduced by @pacospace in #3 . Once that pr is merged I'll rebase ontop of that, until then please review only the portions relevant to the devsecops gropu (i.e. wg section in sigs.yaml and the devsecops charter.md).

### CODE REVIEW

**Comment by HumairAK** [comment]
@durandom @goern @harshad16

File: wg-odhDevSecOps/charter.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# WG DevSecOps
```
Comment (durandom) [inline wg-odhDevSecOps/charter.md:1]: naming conventions. `wg-odhDevSecOps/charter.md`  is a mixture of dash and CamelCase. Suggest we stick to dashed naming

Comment (HumairAK) [reply wg-odhDevSecOps/charter.md]: updated

File: sigs.yaml [hunk 2]
```diff
@@ -24,6 +24,33 @@ workinggroups:
     frequency: weekly
     url: meet.google.com/qkt-yacp-wzm
   contact: {}
+- dir: wg-devsecops
```
Comment (goern) [inline sigs.yaml:27]: looks like you used a different directory for the files in this PR?! The dirname in here looks good, please use that :)

**Review by goern — CHANGES_REQUESTED** [review]

File: sigs.yaml (hunk 2 above)
Comment (HumairAK) [reply sigs.yaml:27]: done

**Comment by goern** [comment]
@HumairAK I figured https://github.com/open-services-group/community/pull/7#issuecomment-1010062942 could you add that too? thx!!

**Comment by HumairAK** [comment]
@goern added, is that what you had in mind?

**Comment by goern** [comment]
:cupid:  it! 
/approve

**Pull request merged**